docs/process/v2_to_v3_environment_variables.md

```diff
@@ -0,0 +1,54 @@
+## Migrating PMM v2 environment variables to v3
+We've renamed some environment variables used by PMM for consistency.
+Below is a list of affected variables and their new names.
+
+| PMM 2                                        | PMM 3                                    | Comments |
+|----------------------------------------------|------------------------------------------|------|
+| `PERCONA_PLATFORM_API_TIMEOUT`               | `PMM_PERCONA_PLATFORM_API_TIMEOUT`       |      |
+| `ENABLE_RBAC`                                | `PMM_ENABLE_RBAC`                        |      |
+| `PERCONA_TEST_INTERFACE_TO_BIND`             | `PMM_INTERFACE_TO_BIND`                  |      |
+| `PERCONA_TEST_PLATFORM_PUBLIC_KEY`           | `PMM_PERCONA_TEST_PLATFORM_PUBLIC_KEY`        |      |
+| `PERCONA_TEST_PLATFORM_ADDRESS`              | `PMM_PERCONA_TEST_PLATFORM_ADDRESS`           |      |
+| `PERCONA_TEST_PLATFORM_INSECURE`             | `PMM_PERCONA_TEST_PLATFORM_INSECURE`          |      |
+| `DISABLE_UPDATES`                            | `PMM_DISABLE_UPDATES`                    |      |
+| `DISABLE_TELEMETRY`                          | `PMM_DISABLE_TELEMETRY`                  |      |
+| `DISABLE_ALERTING`                           | `PMM_DISABLE_ALERTING`                   |      |
+| `METRICS_RESOLUTION`                         | `PMM_METRICS_RESOLUTION`                 |      |
+| `METRICS_RESOLUTION_MR`                      | `PMM_METRICS_RESOLUTION_MR`              |      |
+| `METRICS_RESOLUTION_HR`                      | `PMM_METRICS_RESOLUTION_HR`              |      |
+| `METRICS_RESOLUTION_LR`                      | `PMM_METRICS_RESOLUTION_LR`              |      |
+| `DATA_RETENTION`                             | `PMM_DATA_RETENTION`                     |      |
+| `ENABLE_AZUREDISCOVER`                       | `PMM_ENABLE_AZUREDISCOVER`               |      |
+| `PERCONA_TEST_PMM_CLICKHOUSE_DATABASE`       | `PMM_CLICKHOUSE_DATABASE`                |      |
+| `PERCONA_TEST_PMM_CLICKHOUSE_BLOCK_SIZE`     | `PMM_CLICKHOUSE_BLOCK_SIZE`              |      |
+| `PERCONA_TEST_PMM_CLICKHOUSE_ADDR`           | `PMM_CLICKHOUSE_ADDR`                    |      |
+| `PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE`      | `PMM_CLICKHOUSE_POOL_SIZE`               |      |
+| `DISABLE_UPDATES`                            | `PMM_DISABLE_UPDATES`                    |      |
+| `ENABLE_VM_CACHE`                            | `PMM_ENABLE_VM_CACHE`                    |      |
+| `DISABLE_BACKUP_MANAGEMENT`                  | `PMM_DISABLE_BACKUP_MANAGEMENT`          |      |
+| `NO_PROXY`                                   | `PMM_NO_PROXY`                           |      |
+| `HTTP_PROXY`                                 | `PMM_HTTP_PROXY`                         |      |
+| `HTTPS_PROXY`                                | `PMM_HTTPS_PROXY`                        |      |
```

Comment:
are these PMM environment variables? They can be system level env vars, so we should ignore them.